projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cds/profile-tag.ts

```diff
@@ -1,6 +1,19 @@
 import { clickAllowAllFromBanner } from '../../../helpers/anonymous-consents';
 
+enum EventNames {
+  KEYWORD_SEARCH = 'KeywordSearch',
```

Comment:
As we're renaming this event from to just _SearchEvent_, let's be consistent and apply the change here as well.

Reply:
In our world we always use the term KeywordSearch. Is it a problem or can we leave it like this?